Jasmine/Tetris/Models/TetrisGrid.swift

```diff
@@ -0,0 +1,30 @@
+/// Manages the tiles on the grid
+class TetrisGrid {
+
+    private var tiles: [Coordinate: String] = [:]
+
+    func getTileText(at coordinate: Coordinate) -> String? {
+        return tiles[coordinate]
+    }
+
+    func hasTile(at coordinate: Coordinate) -> Bool {
+        return tiles[coordinate] != nil
+    }
+
+    func addTile(at coordinate: Coordinate, tileText: String) {
+        tiles[coordinate] = tileText
+    }
+
+    func removeTile(at coordinate: Coordinate) -> String? {
+        guard let removedValue = tiles.removeValue(forKey: coordinate) else {
+            return nil
+        }
+        return removedValue
+    }
+
+    func removeTiles(at coordinates: Set<Coordinate>) {
```

Comment:
https://developer.apple.com/reference/swift/array/1641390-remove
https://developer.apple.com/reference/swift/array/1641473-removeall
In many common libraries, single removal returns the removed value while multiple removal does not (because it's common that you don't need the set of removed values in multiple removal)

But if after we refactor this and some game need the set of removed values, we can add it in.